example/gluon/lipnet/BeamSearch.py

```diff
@@ -0,0 +1,168 @@
+# Licensed to the Apache Software Foundation (ASF) under one
```

Comment:
can we add shebang with python version? Preferably python3? thanks.

Reply:
Added. Thanks!